modules/ROOT/pages/enable-inbound-traffic-self.adoc

```diff
@@ -1,3 +1,35 @@
-= Enable Inbound Traffic with a Domain
+= Configure Domains and Inbound Traffic (RTF Self-managed Kuberenetes)
+
+To enable external application to connect to Runtime Fabric on self-managed Kubernetes you must provide an Ingress controler that is compatible with your Kubernetes environment. Runtime fabric uses the Kubernetes controller to handle routing and load balancing functionality. 
```

Comment:
Last sentence: "The ingress controller is used to handle routing and load balancing requests to applications."
Minor typo: "controler"

Reply:
Rewrote as: To enable external application to connect to Runtime Fabric on Self-Managed Kubernetes you must provide an ingress controller that is compatible with your Kubernetes environment. Runtime fabric uses the ingress controller to route and load balance requests to an application.